<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Store</class>
 <widget class="QWidget" name="Store">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4,3,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>最近在玩游戏列表</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="play_current_list"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>新游戏发布</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="new_game_list">
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="8,2">
           <item>
            <widget class="QWidget" name="game_a_widget" native="true">
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>170</y>
                <width>54</width>
                <height>12</height>
               </rect>
              </property>
              <property name="text">
               <string>游戏A图片</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gameA_lable">
             <property name="text">
              <string>免费试玩/售价：100积分</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="8,2">
           <item>
            <widget class="QWidget" name="game_a_widget_2" native="true">
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>170</y>
                <width>54</width>
                <height>12</height>
               </rect>
              </property>
              <property name="text">
               <string>游戏B图片</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gameB_label">
             <property name="text">
              <string>免费试玩/售价：50积分</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>福利中心</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="welfare_center_list"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>积分活动</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="game_point_list"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
